src/tools/dev/scripts/bv_support/bv_ispc.sh

```diff
@@ -45,15 +45,16 @@ function bv_ispc_info
         export ISPC_FILE=${ISPC_FILE:-"ispc-v${ISPC_VERSION}-osx.tar.gz"}
         export ISPC_URL=${ISPC_URL:-"http://sdvis.org/ospray/download/dependencies/osx/"}
         export ISPC_INSTALL_DIR_NAME=ispc-v$ISPC_VERSION-osx
+        export ISPC_SHA256_CHECKSUM="aa307b97bea67d71aff046e3f69c0412cc950eda668a225e6b909dba752ef281"
     else
         export ISPC_FILE=${ISPC_FILE:-"ispc-v${ISPC_VERSION}-linux.tar.gz"}
         export ISPC_URL=${ISPC_URL:-"http://sdvis.org/ospray/download/dependencies/linux/"}
         export ISPC_INSTALL_DIR_NAME=ispc-v$ISPC_VERSION-linux
+        export ISPC_SHA256_CHECKSUM="5513fbf8a2f6e889232ec1e7aa42f6f0b47954dcb9797e1e3d5e8d6f59301e40"
     fi
     export ISPC_COMPATIBILITY_VERSION=${ISPC_COMPATIBILITY_VERSION:-"${ISPC_VERSION}"}
     export ISPC_BUILD_DIR=${ISPC_BUILD_DIR:-"${ISPC_VERSION}"}
     export ISPC_MD5_CHECKSUM="0178a33a065ae65d0be00be23871cf9f"
```

Comment:
You should either remove the md5 checksum or put mac and linux specific ones under the if block above.

Reply:
I generated md5 checksums for OSX. Assuming that the original checksum is for linux.